code/game/objects/items/food/soup.dm

```diff
@@ -155,7 +155,7 @@
 	icon_state = "milosoup"
```

Comment:
I assume this is supposed to be miso? Why the fuck is it milo?

Reply:
milo soup